drink/src/mock/mocking_api.rs

```diff
@@ -0,0 +1,61 @@
+use crate::{
+    contract_api::ContractApi,
+    mock::ContractMock,
+    runtime::{AccountIdFor, Runtime},
+    Sandbox, DEFAULT_GAS_LIMIT,
+};
+
+/// Interface for basic mocking operations.
+pub trait MockingApi<R: Runtime> {
+    /// Deploy `mock` as a standard contract. Returns the address of the deployed contract.
+    fn deploy_mock(&mut self, mock: ContractMock) -> AccountIdFor<R>;
+
+    /// Mock part of an existing contract. In particular, allows to override real behavior of
+    /// deployed contract's messages.
+    fn mock_existing_contract(&mut self, mock: ContractMock, address: AccountIdFor<R>);
+}
+
+impl<R: Runtime> MockingApi<R> for Sandbox<R> {
+    fn deploy_mock(&mut self, mock: ContractMock) -> AccountIdFor<R> {
```

Comment:
As mentioned in the comment under the example - can we rename to `deploy`?

Reply:
:white_check_mark: